docs/getting-started/installation.md

```diff
@@ -20,7 +20,9 @@ The soda-sql CLI requires the following dependencies to be installed on your sys
 - Python >=3.7 <3.9
 - Pip >=21.0
 - postgresql-libs (`libpq-dev` in Debian/Ubuntu, `libpq-devel` in CentOS, `postgresql` on MacOSX)
-- _Linux only:_ `libssl-dev` and `libffi-dev` (`libffi-devel` and `openssl-devel` in CentOS)
+- _Linux only:_ 
+  + On Debian Buster: `apt-get install g++ unixodbc-dev python3-dev libssl-dev libffi-dev`
+  + On CentOS 8: `yum install gcc-c++ unixODBC-devel python38-devel libffi-devel openssl-devel`
 
```

Comment:
We probably need to point to https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver15 for macOS here, unixodbc wasn't working with SQLServer

Reply:
I think that page is a must read if they are actually trying out MSSQL in runtime, right? I think this above is enough to do a `pip install soda-sql`, but MSSQL will not work until instructions on that page are followed?

Reply:
Yes, the install will run but scan/analyze will fail. Not sure if that gives a reasonable error message. For me it didn't work until I installed odbc driver, even though pyodbc claimed otherwise.

Reply:
Alright, so if it’s indeed runtime then we should add here “for MSSQL you need to follow this link”, and not ask each user to install those things needs for the MSSQL, agreed?